web/rootfs/defaults/meet.conf

```diff
@@ -35,8 +35,8 @@ location ~ ^/(libs|css|static|images|fonts|lang|sounds|connection_optimization|.
 }
 
 # colibri (JVB) websockets
-location ~ ^/colibri-ws/default-id/(.*) {
-    proxy_pass http://jvb.meet.jitsi:9090/colibri-ws/default-id/$1$is_args$args;
+location ~ ^/colibri-ws/([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+)/(.*) {
```

Comment:
Since this container might be used standalone, let's have a catch-all here. It would be possible to use domains and connect over the public internet for example.